pyhub-installer/.github/workflows/release.yml

```
@@ -0,0 +1,98 @@
+      with:
+        name: ${{ matrix.goos }}-${{ matrix.goarch }}
+        path: dist/*
+
+  release:
+    name: Create Release
+    if: startsWith(github.ref, 'refs/tags/')
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Download all artifacts
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)